core/block/editor/template/collection.go

```diff
@@ -9,68 +11,137 @@ import (
 	"github.com/anyproto/anytype-heart/util/pbtypes"
 )
 
-func MakeCollectionDataviewContent() *model.BlockContentOfDataview {
-	relations := []*model.RelationLink{
-		{
-			Format: model.RelationFormat_shorttext,
-			Key:    bundle.RelationKeyName.String(),
-		},
+const (
+	CollectionStoreKey = "objects"
+	defaultViewName    = "All"
+)
+
+var (
+	defaultDataviewRelations = []domain.RelationKey{
+		bundle.RelationKeyName,
+		bundle.RelationKeyCreatedDate,
+		bundle.RelationKeyCreator,
+		bundle.RelationKeyLastModifiedDate,
+		bundle.RelationKeyLastModifiedBy,
+		bundle.RelationKeyLastOpenedDate,
+		bundle.RelationKeyBacklinks,
 	}
-	viewRelations := []*model.BlockContentDataviewRelation{
-		{
-			Key:       bundle.RelationKeyName.String(),
-			IsVisible: true,
-		},
+
+	defaultCollectionRelations = []domain.RelationKey{
+		bundle.RelationKeyName,
+		bundle.RelationKeyType,
+		bundle.RelationKeyCreatedDate,
+		bundle.RelationKeyCreator,
+		bundle.RelationKeyLastModifiedDate,
+		bundle.RelationKeyLastModifiedBy,
+		bundle.RelationKeyLastOpenedDate,
+		bundle.RelationKeyBacklinks,
+		bundle.RelationKeyTag,
+		bundle.RelationKeyDescription,
 	}
-	for _, relKey := range DefaultDataviewRelations {
-		if pbtypes.HasRelationLink(relations, relKey.String()) {
-			continue
+
+	defaultVisibleRelations = []domain.RelationKey{
+		bundle.RelationKeyName,
+		bundle.RelationKeyType,
+	}
+)
+
+func MakeDataviewContent(isCollection bool, ot *model.ObjectType, relLinks []*model.RelationLink) *model.BlockContentOfDataview {
+	var (
+		defaultRelations = defaultCollectionRelations
+		visibleRelations = defaultVisibleRelations
+		sorts            = DefaultLastModifiedDateSort()
+	)
+
+	if isCollection {
```

Comment:
L56-66 - may be extract to separate function? Seems like it make logic of `MakeDataviewContent` complicated